books/migrations/0003_change_author_table_&apply_data_migration.py

```diff
@@ -0,0 +1,63 @@
+# Generated by Django 3.2.5 on 2021-07-13 20:21
```

Comment:
`&` is not a good idea to use in file names

Reply:
great. will change